profile.html

```diff
@@ -1054,7 +1054,7 @@
                                                   <div class="_1sgu3hhe">
                                                     <input
                                                       name="work"
-                                                      class="_1otaoi4o"
+                                                      class="form__input--small"
```

Comment:
```suggestion
                                                      class="form__input"
```

bago nyo po irename to pwede paki ayos narin ng mga dating form__input gawing form-floating

Reply:
What do you mean by this po?
Meron na kasi dating form__input